flang/test/Integration/fp-convert.f90

```diff
@@ -0,0 +1,236 @@
+! RUN: %flang -funsigned %s -o %t && %t | FileCheck %s
```

Comment:
Lit tests should not contain end-to-end tests, they belong to https://github.com/llvm/llvm-test-suite/tree/main/Fortran/UnitTests (this is a "whole program test" for https://llvm.org/docs/TestingGuide.html#llvm-testing-infrastructure-organization).

It is a great end to end test, so I would encourage you to add it there.

Reply:
Thanks! Do you think the small changes I made to `flang/test/Fir/convert-to-llvm.fir` are enough for this PR, then? Those are the only differences we should see at the fir level.